packages/template-mrt-reference-app/app/ssr.js

```diff
@@ -264,6 +264,11 @@ const {handler, app, server} = runtime.createHandler(options, (app) => {
     app.get('/headers', headerTest)
     app.get('/isolation', isolationTests)
 
+    // Ensure response headers can contain commas
+    app.get('/redirect-with-commas', (req, res) => {
+        res.redirect('/22,testing,cc.html')
+    })
```

Comment:
As @bendvc suggested, would you consider making this redirect dynamic so we could test all manner of redirects in the future even if our specific test case right now involves commas?

```suggestion
    app.get('/redirect/*', (req, res) => {
        const path = req.params[0];
        res.redirect(`/redirected/${path}`);
    });
```

This way you can request:

```
/redirect/my,redirect,with,commas
```

But all:

```
/redirect/my;redirect;with;colons
```